CalcPi.java

```diff
@@ -1,6 +1,17 @@
 // Computes an approximation of PI.
 public class CalcPi {
 	public static void main(String [] args) { 
-	    // Replace this comment with your code
+	int num = Integer.parseInt(args[0]);
+	int i = 0;
+	double sum = 0.0;
+
+	for (i = 0; i < num; i++){
+	double term = Math.pow(-1, i) / (2*i + 1);
```

Comment:
1. indentation 
2. spacing
3. declare i in for line, no usage outside the loop